src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/RequestExecutionQueue.cs

```diff
@@ -156,6 +170,8 @@ private async Task ProcessQueueAsync()
         ILspServices? lspServices = null;
         try
         {
+            var pendingTasks = new ConcurrentDictionary<Task, CancellationTokenSource?>();
```

Comment:
Perhaps name to `concurrentlyExecutingTasks`.  pending just soudns weird to me.

Reply:
Naming is hard :)